packages/components/pages/f-contact-preferences/src/components/ContactPreferences.vue

```diff
@@ -17,40 +17,29 @@
                     </h2>
                     <fieldset
                         :class="$style['c-contactPreferences-fieldset']">
-                        <label>
-                            <input
-                                type="checkbox"
-                                :data-test-id="`contact-preferences-${key}-email-checkbox`"
-                                :disabled="!isEmailEnabled"
-                                :checked="emailValue"
-                                @change="editPreferenceValue(key, Object.keys({ emailValue })[0], $event.target.checked)">
-                            <span
-                                :class="{
-                                    [$style['c-contactPreferences-labelText--disabled']]: !isEmailEnabled
-                                }">
-                                {{ $t(`${key}.email`) }}
-                                <template v-if="$te(`${key}.emailDescription`)">
-                                    <br>({{ $t(`${key}.emailDescription`) }})
-                                </template>
-                            </span>
-                        </label>
-                        <label>
-                            <input
-                                type="checkbox"
-                                :data-test-id="`contact-preferences-${key}-sms-checkbox`"
-                                :disabled="!isSmsEnabled"
-                                :checked="smsValue"
-                                @change="editPreferenceValue(key, Object.keys({ smsValue })[0], $event.target.checked)">
-                            <span
-                                :class="{
-                                    [$style['c-contactPreferences-labelText--disabled']]: !isSmsEnabled
-                                }">
-                                {{ $t(`${key}.sms`) }}
-                                <template v-if="$te(`${key}.smsDescription`)">
-                                    <br>({{ $t(`${key}.smsDescription`) }})
-                                </template>
-                            </span>
-                        </label>
+                        <f-form-field
+                            :class="$style['c-contactPreferences-form-field']"
```

Comment:
To match the naming CSS naming convention, this should be `formField` here (and below where it's used too)

```suggestion
                            :class="$style['c-contactPreferences-formField']"
```